zilliqa/src/scilla.rs

```diff
@@ -923,10 +923,48 @@ impl ActiveCall {
         }
 
         if ignore_value {
-            // We only supporting deleting a single value of a map.
-            assert_eq!(indices.len(), depth);
```

Comment:
This property is relied upon in other places too. Now it doesn't hold, we can't make these assumptions any more and need to implement these methods too:
https://github.com/Zilliqa/zq2/blob/69aa81e4ba4bdf8496a3b89a5e3b1541064e870e/zilliqa/src/exec.rs#L1256
https://github.com/Zilliqa/zq2/blob/69aa81e4ba4bdf8496a3b89a5e3b1541064e870e/zilliqa/src/exec.rs#L1297